### DISCUSSION TITLE
How to use only notification module ?

### DISCUSSION THREAD

**Discussion by jigujigu** [filamentphp/filament#4561] [Help] [answered]
Hello, 
Is it  possible to use  only notification module in liveware component?

For testing propose, I create new laravel project and install livewire. Then I create counter component like on document.

I install the filament notification base on this instructions:

composer require filament/notifications
php artisan notifications:install

then I add notification code in counter component,  But why notification not showing ?

Is there any step I miss ?? 
Thank You
```
<?php

namespace App\Http\Livewire;

use Filament\Notifications\Notification;
use Livewire\Component;

class Counter extends Component
{
    public $count = 0;

    public function increment()
    {
        $this->count++;

        Notification::make()
        ->title('Saved successfully')
        ->success()
        ->send();
    }

    public function render()
    {
        return view('livewire.counter');
    }
}
```

**Comment by zepfietje** [comment, marked as answer]
Make sure to follow all installation steps (including setting up the script and preparing your layout): https://filamentphp.com/docs/2.x/notifications/installation.
